code/game/say.dm

```diff
@@ -67,8 +67,16 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	if(istype(D) && D.display_icon(src))
 		languageicon = "[D.get_icon()] "
 
-	return "[spanpart1][spanpart2][CMSG_FREQPART][languageicon][CMSG_JOBPART][namepart][endspanpart][messagepart]"
+	return "[spanpart1][spanpart2][CMSG_FREQPART][languageicon][CMSG_JOBPART][compose_name_href(namepart)][endspanpart][messagepart]"
 
+/**
+	Allows us to wrap the name for specific cases like AI tracking or observer tracking
+
+	Arguments
+	- name {string} the name of the mob to modify.
+*/
```

Comment:
```suggestion
/**
  *Allows us to wrap the name for specific cases like AI tracking or observer tracking
  *
  * *Arguments
  * * name {string} the name of the mob to modify.
*/
```

Reply:
The extra * will actually break the markup in the end.

With how i have it now the arguments will parse into a list